src/app/features/dialogs/switch-account-dialog/switch-account-dialog.tsx

```diff
@@ -43,6 +43,7 @@ export const SwitchAccountDialog = memo(({ isShowing, onClose }: DialogProps) =>
   if (!isShowing) return null;
 
   const accountNum = stacksAddressesNum || btcAddressesNum;
+  const maxAccountsShown = accountNum > 10 ? 10 : accountNum;
```

Comment:
@kyranjamie - you were right about this and I had naively set the list to have a generated height based on all of the accounts. This change limits it to only allow the height of 10 accounts to be rendered. 

When I test I still see more than 10 rendered so I need to figure that out. I see `15` items rendered when I can only see 9 in the viewport. 

The reason I changed this code was so that the UI could smoothly adjust and increase the list height on smaller width screens. Adding this code impacts that but I am close to having a better fix in: https://github.com/leather-wallet/extension/pull/5171 

I just thought it best to get something mergeable for this as it was a bad bug on `dev` and a release blocker